### DISCUSSION TITLE
Can't open capture devices - COM port in use error

### DISCUSSION THREAD

**Discussion by jammer133** [bengtmartensson/IrScrutinizer#499] [Hardware issues] [answered]
I have used IRScrutinizer successfully in the past with both the Arduino IR Widget and the Arduino GIRS interface.  It's been awhile since I've used it, but now I can't open either device and get a COM port in use error for both.  Interestingly, I can still use the Arduino IR Wdiget with IRScope without problems on the same COM port.  I've tried it on two different computers with same result.  I'm not sure if it's a Windows 11 issue maybe due to a recent update but just checking if there are known problems like this.

OS is Windows 11 with 22H2 version on both computers.

Any ideas?

Jay

**Comment by bengtmartensson** [comment]
I am sorry, right now I do not know. I have opened an [issue in nrjavaserial](https://github.com/NeuronRobotics/nrjavaserial/issues/245)  (which is the serial library used). Presently, I do not have access to Windows 11 here, sorry.

**Comment by jammer133** [comment, marked as answer]
Hi - I think I have resolved it.  It seems to be due to an updated CH340 driver for the Arduino.  See this [[link](url).](https://forum.arduino.cc/t/possible-bug-with-device-driver-for-usb-port/1155183).

I rolled back the driver to 3.7.2022 and IRScrutinizer communicated fine again.  I installed the driver from the download and then rolled back using the device manager, so hopefully Windows 11 will not update this driver in the future.
